src/verify.cc

```diff
@@ -135,5 +136,19 @@ Status verifyJwt(const Jwt& jwt, const Jwks& jwks, int64_t now) {
                          : Status::JwksKidAlgMismatch;
 }
 
+Status verifyJwt(const Jwt& jwt, const Jwks& jwks,
+                 const std::vector<std::string>& audiences) {
+  return verifyJwt(jwt, jwks, audiences, absl::ToUnixSeconds(absl::Now()));
+}
+
+Status verifyJwt(const Jwt& jwt, const Jwks& jwks,
+                 const std::vector<std::string>& audiences, int64_t now) {
+  CheckAudience checker(jwt.audiences_);
```

Comment:
you should use pass in audiences as config_audiences in the constructor.  The subtle different is:  
if config_audience is empty,  not need to check jwt audiences. 

Please add comment on the prototype the behaviors of empty audiences in the function argument.

Reply:
Fixed.